app/layouts/DashboardLayout/ShowEntropyModal.view/ShowEntropyModal.view.tsx

```diff
@@ -104,9 +104,11 @@ const ShowEntropyModal: FC<ShowEntropyModalProps> = ({
                     </Fab>
                   </Box>
                   <Container maxWidth="sm">
-                    <Typography variant="body2" color="textPrimary">
-                      <div style={{ minHeight: 60 }}>{showEntropy ? mnemonic : ''}</div>
-                    </Typography>
+                    <Box minHeight={60}>
```

Comment:
fix issue with <div> nested under <p>

Reply:
youre amazing. that error was such a pita

Reply:
Is this a general thing with MUI's Typography component?

Reply:
it's an html thing. `<p><div><div/></p>` isn't valid html and Typography renders a `<p></p>`